pom.xml

```diff
@@ -70,7 +70,7 @@
             </dependency>
             <dependency>
                 <groupId>com.fasterxml.jackson.module</groupId>
-                <artifactId>jackson-module-afterburner</artifactId>
```

Comment:
`afterburner` results in `IllegalAccessException`s, which were allowed in Java 11, but now stop the build in Java 17.

The description of `jackson-blackbird` is literally "Upgrade your Afterburner for your Java 11+ Environment", so I've replaced afterburner with it.